packages/plugin-persistent-cache/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@vitejs/plugin-persistent-cache",
+  "version": "4.1.1",
+  "license": "MIT",
+  "author": "Guillaume Chau",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "frontend",
+    "vite",
+    "vite-plugin",
+    "cache"
+  ],
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "scripts": {
```

Comment:
I'm guessing we need to copy the `lint`, `format`, and `typecheck` from the main Vite package to be here as well